app/src/main/java/org/xwiki/android/sync/activities/SelectAccountActivity.kt

```diff
@@ -28,7 +29,9 @@ class SelectAccountActivity : BaseActivity(), AccountClickListener {
                 userAccountsRepo.getAll(),
                 this@SelectAccountActivity
             )
-            binding.lvAvailableAccounts.adapter = adapter
+            val recyclerView = binding.lvAvailableAccounts
+            recyclerView.layoutManager = LinearLayoutManager(this@SelectAccountActivity)
```

Comment:
I prefer to set up layout manager from xml (like it described here https://stackoverflow.com/a/35680716 )